Cargo.toml

```diff
@@ -104,6 +104,8 @@ rand = "0.9"
 rstest = "0.23"
 regex = "1.11"
 anyhow = { version = "1.0", default-features = false }
+tempfile = "3.17.1"
```

Comment:
this should just be in `dev-dependencies` I think, since we only use it for tests